### ISSUE TITLE
bdd100k/bdd100k/MOT/tid_num.json,tid_numval.json

### ISSUE THREAD

**Issue by ws-26-ws** [supperted825/FairMOT-X#13]
Hello, I want to ask what is in the above two json files and how to obtain them

**Comment by supperted825** [comment]
Hi, these files are counting the number of unique track IDs in the dataset that you use for training and validation respectively. You can obtain them by doing a count of your labels when you first load in your dataset. But this process takes awhile, so caching it to JSON can let you skip this process especially when you are debugging. Please see [here](https://github.com/supperted825/FairMOT-X/blob/42ff726dfb1e19545f30d0a77039f323c663f919/src/lib/datasets/yolomot.py#L353).

**Comment by ws-26-ws** [comment]
I am so happy to receive your reply. Thank you for replying to my questions in your busy schedule. I look forward to the next learning exchange with you.




王帅
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "supperted825/FairMOT-X"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月13日(星期四) 下午3:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [supperted825/FairMOT-X] bdd100k/bdd100k/MOT/tid_num.json,tid_numval.json (Issue #13)





 
Hi, these files are counting the number of unique track IDs in the dataset that you use for training and validation respectively. You can obtain them by doing a count of your labels when you first load in your dataset. But this process takes awhile, so caching it to JSON can let you skip this process especially when you are debugging. Please see here.

**Comment by ws-26-ws** [comment]
Hello, I have another question, because I'm still a beginner and I'm usually a fairmot. How many rounds should I train for tracking training?




王帅
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "supperted825/FairMOT-X"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月13日(星期四) 下午3:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [supperted825/FairMOT-X] bdd100k/bdd100k/MOT/tid_num.json,tid_numval.json (Issue #13)





 
Hi, these files are counting the number of unique track IDs in the dataset that you use for training and validation respectively. You can obtain them by doing a count of your labels when you first load in your dataset. But this process takes awhile, so caching it to JSON can let you skip this process especially when you are debugging. Please see here.

**Comment by supperted825** [comment]
It depends on your dataset. But for FairMOT with YOLOX, for me around 20 epochs is enough on BDD100K (very big dataset). It will depend on your learning rate / learning rate decay also.

**Comment by ws-26-ws** [comment]
Hello, boss, I'm really sorry to bother you again. Some problems occurred during the demo after the training, as shown in the figure below.&nbsp;How to solve this problem?&nbsp;And what is the size of the photo entered in YOYOMOT (576，1024),&nbsp;instead of 1024 and 576?






------------------ 原始邮件 ------------------
发件人:                                                                                                                        "supperted825/FairMOT-X"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月13日(星期四) 下午3:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [supperted825/FairMOT-X] bdd100k/bdd100k/MOT/tid_num.json,tid_numval.json (Issue #13)





 
It depends on your dataset. But for FairMOT with YOLOX, for me around 20 epochs is enough on BDD100K (very big dataset). It will depend on your learning rate / learning rate decay also.
